versions/3.0.md

```diff
@@ -1242,6 +1243,7 @@ Field Name | Type | Description
 <a name="schemaXml"></a>xml | [XML Object](#xmlObject) | This MAY be used only on properties schemas. It has no effect on root schemas. Adds Additional metadata to describe the XML representation format of this property.
 <a name="schemaExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this schema. 
 <a name="schemaExample"></a>example | Any | A free-form property to include a an example of an instance for this schema.
+<a name="schemaDeprecated"></a> deprecated | `boolean` | Specifies that a schema is deprecated and should be transitioned out of usage.
```

Comment:
> but maybe it is better to make this a string property, with the value being the deprecation documentation?

@ePaul It was discussed here: https://github.com/OAI/OpenAPI-Specification/issues/540
And it's a slippery slope since next thing people ask to implement date of deprecation.
IMHO it's very simple rule of thumb if you can programmatically act based on field value it's one thing.
But if it text for a user to read than description can be used for that. 

One thing that could be added to Swagger spec, is recommendation to add full content of description field into the text of deprecation warning.